TicketMate.Persistence.Tests/DataRequestTests/ProjectTests/GetProjectByGuidTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketMate.Persistence.DataRequestObjects.ProjectRequests;
+
+namespace TicketMate.Persistence.Tests.DataRequestTests.ProjectTests
+{
+    public class GetProjectByGuidTests : BaseDataRequestTest
+    {
+        [Fact]
+        public async Task GetProjectByGuid_ProjectNotExesting_ShouldReturn_Null()

```

Comment:
your other test has _Given_ in the name, so maybe add here also? Other than that everything looks great to me!